Community/alumni.md

```diff
@@ -240,20 +289,17 @@ Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur
 Augur is fully functional prototyping web stack for CHAOSS metrics. It provides structured data mined from git repositories using a plugin architecture that incorporates other open source metrics projects like Facade and FOSSology. The main aim of this project is to extend Augur’s functionality by implementing Risk and Growth-Maturity-Decline CHAOSS metrics and use cases with a focus on the open source community manager use case. This project, with a focus on the community manager use case, will allow open source community managers to leverage Risk and Growth-Maturity-Decline metrics to better manage their communities and projects.
 
 
-#### Blog posts
-
-* [Weekly Posts](https://parthsharma2.github.io/posts)
-
-
-#### Social Media
+#### Links
 
+* [Project Link on GSoC Website](https://summerofcode.withgoogle.com/archive/2019/projects/5952207168471040/)
+* [Blog Posts](https://parthsharma2.github.io/posts)
 * [LinkedIn](https://www.linkedin.com/in/parthsharma2)
 * [Twitter](https://twitter.com/parthshar2)

```

Comment:
remove twitter and linkedin?

Reply:
@vchrombie should i remove all twitter and linkedin links and keep only github links is that what you are suggesting?

Reply:
Yeah, I was thinking it would be good if we maintain consistency on the page by having the same set of links.
WDYT?

Reply:
@vchrombie Ok I am doing it